website/src/pages/features/index.tsx

```diff
@@ -48,35 +67,59 @@ function FeatureManageImages() {
 <div className="lg:flex-grow md:w-1/2 lg:pl-24 md:pl-16 flex flex-col md:items-start md:text-left items-center text-center">
   <h1 className="title-font sm:text-4xl text-3xl mb-4 font-medium text-gray-900 dark:text-white">Build, Pull and Push images</h1>
 
-<p className="leading-relaxed list-item">Build images from the tool.</p>
-<p className="leading-relaxed list-item">Pull and push images by managing registries.</p>
-<p className="leading-relaxed list-item">Run containers from these images</p></div>
+<p className="leading-relaxed list-item dark:text-gray-200 text-gray-800">Build images from the tool.</p>
+<p className="leading-relaxed list-item dark:text-gray-200 text-gray-800">Pull and push images by managing registries.</p>
+<p className="leading-relaxed list-item dark:text-gray-200 text-gray-800">Run containers from these images</p></div>
 
 </div>
 </section>);
 
 }
 
-
-
-function FeatureManageResources() {
+function FeatureManagementFromTrayIcon() {
   return (
     <div>
-      <section className="text-gray-600 dark:bg-zinc-800 dark:text-gray-40 body-font">
+      <section className="text-gray-600 bg-zinc-200 dark:bg-zinc-800 dark:text-gray-400 body-font">
         <div className="container mx-auto flex px-5 py-24 md:flex-row flex-col items-center">
           <div className="lg:flex-grow md:w-1/2 lg:pr-24 md:pr-16 flex flex-col md:items-start md:text-left mb-16 md:mb-0 items-center text-center">
             <h1 className="title-font sm:text-4xl text-3xl mb-4 font-medium text-gray-900 dark:text-white">
-              Manage Podman resources
+              Management from the tray icon
               <br className="hidden lg:inline-block" />
             </h1>
             <p className="leading-relaxed list-item dark:text-gray-200 text-gray-800">
-              View allocated memory, CPU and storage.
+              Check status and start/stop container engines.
             </p>
             <p className="leading-relaxed list-item dark:text-gray-200 text-gray-800">
-              Create new machine if needed
+              Create new machine if needed.
+            </p>
+            <p className="leading-relaxed list-item dark:text-gray-200 text-gray-800">
+              Start or stop Podman machines directly from the tray icon.
```

Comment:
Maybe we can merge the 2 bullets on machines into a single one? wdyt? 
Add a third one about the ability to "quickly check activity status"?

Reply:
Sure, that helps us give out more information. I will make the change.